docs/src/windows.md

```diff
@@ -93,4 +93,12 @@ npm start
 
 ```
 npm watch
+```
+
+- If W10's Linux Bash stops resolving domain names, here's a fix ([source](https://github.com/microsoft/WSL/issues/3268#issuecomment-485096972)):
+
+```
+sudo rm /etc/resolv.conf
+echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf
```

Comment:
Maybe we could use a placeholder here, to avoid favoring any particular DNS server :)